internal/rbd/encryption.go

```diff
@@ -463,6 +465,34 @@ func (rv *rbdVolume) RotateEncryptionKey(ctx context.Context) error {
 		return errors.New("key rotation not supported for unencrypted device")
 	}
 
+	// Call open Ioctx to create a new ioctx object
+	// if the obj already exists, no error is returned
+	err = rv.openIoctx()
+	if err != nil {
+		log.ErrorLogMsg("failed to open ioctx, err: %v", err)
```

Comment:
no need to log here

Reply:
Should I remove the one at line 488 as well?

Reply:
i think just here is fine

Reply:
we are not logging anywhere in this function, lets do the same for line 488 as well

Reply:
There is one debug log once we acquire the lock.

Reply:
that should be fine, we need that for debugging

Reply:
Done.